federation-js/src/service/__tests__/printSupergraphSdl.test.ts

```diff
@@ -311,11 +308,9 @@ describe('printSupergraphSdl', () => {
     `);
   });
 
-  it('prints a fully composed schema without @tag or @inaccessible correctly', () => {
+  it('prints a fully composed schema without @tag orcorrectly', () => {
```

Comment:
Fyi, you left the `or` mistakenly, but I took the liberty to ninja-merge the fix in https://github.com/apollographql/federation/commit/e9e9a66d239c05123044edc14d79f60013abcf27.

Reply:
Thanks!